<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>650</height>
   </rect>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuedrt">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPen_color"/>
    <addaction name="actionFill_color"/>
    <addaction name="actionPen_size"/>
    <addaction name="separator"/>
    <addaction name="actionResize"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuDraw">
     <property name="title">
      <string>Draw...</string>
     </property>
     <addaction name="actionCircle"/>
     <addaction name="actionRectangle"/>
     <addaction name="actionTriangle"/>
    </widget>
    <addaction name="menuDraw"/>
    <addaction name="separator"/>
    <addaction name="actionFill_with_coor"/>
    <addaction name="actionErase"/>
    <addaction name="separator"/>
    <addaction name="actionZoom_2"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_PaintQT"/>
   </widget>
   <addaction name="menuedrt"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAdd"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionZoom"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_2"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCircle"/>
   <addaction name="separator"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionTriangle"/>
   <addaction name="separator"/>
   <addaction name="actionLine"/>
   <addaction name="separator"/>
   <addaction name="actionFilledCircle"/>
   <addaction name="separator"/>
   <addaction name="actionFilledRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionFilledTriangle"/>
   <addaction name="separator"/>
   <addaction name="actionErase"/>
   <addaction name="separator"/>
   <addaction name="actionPen_color"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/floppy-disk.png</normaloff>:/images/floppy-disk.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/add-file.png</normaloff>:/images/add-file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPen_color">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/color-wheel.png</normaloff>:/images/color-wheel.png</iconset>
   </property>
   <property name="text">
    <string>Pen color</string>
   </property>
  </action>
  <action name="actionPen_size">
   <property name="text">
    <string>Pen size</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionFill_with_coor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fill with color</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/eraser.png</normaloff>:/images/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/magnifying-glass (1).png</normaloff>:/images/magnifying-glass (1).png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionResize">
   <property name="text">
    <string>Resize image</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/broom.png</normaloff>:/images/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="toolTip">
    <string>Clear All</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionAbout_PaintQT">
   <property name="text">
    <string>About PaintQT</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rectangle.png</normaloff>:/images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/triangle.png</normaloff>:/images/triangle.png</iconset>
   </property>
   <property name="text">
    <string>Triangle</string>
   </property>
   <property name="shortcut">
    <string>T</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionFill_color">
   <property name="text">
    <string>Fill color</string>
   </property>
  </action>
  <action name="actionZoom_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/magnifying-glass.png</normaloff>:/images/magnifying-glass.png</iconset>
   </property>
   <property name="text">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionFilledCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/circular-filled-shape.png</normaloff>:/images/circular-filled-shape.png</iconset>
   </property>
   <property name="text">
    <string>Filled Circle</string>
   </property>
   <property name="toolTip">
    <string>Filled Circle</string>
   </property>
   <property name="shortcut">
    <string>Shift+C</string>
   </property>
  </action>
  <action name="actionFilledRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/rectangle (1).png</normaloff>:/images/rectangle (1).png</iconset>
   </property>
   <property name="text">
    <string>Filled Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Filled Rectangle</string>
   </property>
   <property name="shortcut">
    <string>Shift+R</string>
   </property>
  </action>
  <action name="actionFilledTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/triangle (1).png</normaloff>:/images/triangle (1).png</iconset>
   </property>
   <property name="text">
    <string>Filled Triangle</string>
   </property>
   <property name="toolTip">
    <string>Filled Triangle</string>
   </property>
   <property name="shortcut">
    <string>Shift+T</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/diagonal-line.png</normaloff>:/images/diagonal-line.png</iconset>
   </property>
   <property name="text">
    <string>Line</string>
   </property>
   <property name="toolTip">
    <string>Line</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
